HelpSource/Classes/Env.schelp

```diff
@@ -67,10 +95,49 @@ table::
 ::
 
 argument::releaseNode
-an link::Classes/Integer:: or nil. The envelope will sustain at the release node until released.
+an link::Classes/Integer:: or nil. The envelope will sustain at the releaseNode until released.
+
+code::
+(
+{
+	EnvGen.kr(
+		Env.new(
+			levels: [0, 1, 0.5, 0], 
+			times: [0.01, 0.01, 0.01],
+			releaseNode: 2 // sustains at level 0.5 until gate is closed
+		), 
+		gate: Trig.kr(Impulse.kr(3), dur: 0.3)
+	);
+}.plot(duration: 1);
+)
+
+::
+
+In this example, the release node is the third node which has a level of 0.5, so the envelope sustains at the level of 0.5, and when the gate is released, the envelope goes to the next node (it reaches 0 in 0.01 second).
 
 argument::loopNode
-an link::Classes/Integer:: or nil. If not nil the output will loop through those nodes starting at the loop node to the node immediately preceding the release node, before back to the loop node, and so on. Note that the envelope only transitions to the release node when released. Examples are below. The loop is escaped when a gate signal is sent, when the output transitions to the release node, as described below.
+an link::Classes/Integer:: or nil. If not nil the output will loop through those nodes starting at the node following the loop node to the release node, before back to the node following the loop node, and so on. The loop is escaped when a gate signal is sent, when the output transitions to the node following the release node, as described below.
+Basically : the release node is part of the loop, the loop node is not, and the envelope will go to the node following the release node when the gate is released.
+You should specify the releaseNode and use a gate signal (and not a trigger signal) to use a looping envelope.
```

Comment:
This is another hard one to describe clearly. This is my suggestion:

"an `Integer` or `nil`. Creates a segment of looping nodes. You must specify a `releaseNode` in order for `loopNode` to have any effect. The `loopNode` is the initial node of the loop and is never repeated. Upon reaching the `releaseNode`, the envelope will move back to the node that immediately follows `loopNode`. The envelope will loop until its gate is closed. When released, a looping envelope will move from its current position to the node that immediately follows `releaseNode` and continue until the end."